README.rst

```diff
@@ -392,7 +392,7 @@ number of user-configurable options:
         "Alert Details" tab on the job results page. Information returned
         includes hex/ASCII output from packets that generated alerts as
         well as "Extra" data from the unified2 file such as "Original
-        Client IP" from packets with "X-Forwared-For" or "True-Client-IP"
+        Client IP" from packets with "X-Forwarded-For" or "True-Client-IP"
```

Comment:
there are some famously misspelled items in http headers, but I don't think this is amongst them